app/models/user.rb

```diff
@@ -1,4 +1,7 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are: , :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] I tried to sign up in your application but I couldn't please it would be better if you fix that in order to make the app work as expected
<img width="1108" alt="Screen Shot 2022-09-28 at 18 43 49" src="https://user-images.githubusercontent.com/61790587/192852572-2ab09f6c-072e-4997-8856-3841cf54a46a.png">

Reply:
Everything seems to be working perfectly well on my devise